app/models/user.rb

```diff
@@ -122,6 +122,13 @@ def suspend!
     OutOfBandSessionAccessor.new(unique_session_id).destroy if unique_session_id
     update!(suspended_at: Time.zone.now, unique_session_id: nil)
     analytics.user_suspended(success: true)
+    email_addresses.flat_map do |email_address|
```

Comment:
we're not using the return value here, or flattening at all right? so we can switch to `each`
```suggestion
    email_addresses.each do |email_address|
```